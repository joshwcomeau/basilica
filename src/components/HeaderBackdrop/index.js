// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import './index.scss';


const HeaderBackdrop = () => {
  const classes = classNames(['header-backdrop']);

  return (
    <div className={classes} />
  );
};

HeaderBackdrop.propTypes = {

};

HeaderBackdrop.defaultProps = {

};

export default HeaderBackdrop;
