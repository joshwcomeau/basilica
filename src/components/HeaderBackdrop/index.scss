@import '../../variables';

.header-backdrop {
  position: fixed;
  z-index: 5;
  height: $header-height;
  top: 0;
  left: 0;
  right: 0;
  max-width: $max-width;
  margin: 0 auto;
  background: $offwhite;

  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
