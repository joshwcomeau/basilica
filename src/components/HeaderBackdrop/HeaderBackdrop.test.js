/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import HeaderBackdrop from './index';

describe('HeaderBackdrop', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HeaderBackdrop />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
