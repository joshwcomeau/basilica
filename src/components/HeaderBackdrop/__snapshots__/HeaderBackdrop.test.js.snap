exports[`HeaderBackdrop renders correctly 1`] = `
<div
  className="header-backdrop" />
`;
